payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/CardNumberController.kt

```diff
@@ -57,8 +57,13 @@ internal class CardNumberController constructor(
         CardBrand.getCardBrands(it).firstOrNull() ?: CardBrand.Unknown
```

Comment:
I think this should be fixed here
```suggestion
        CardBrand.getCardBrands(it).takeIf { it.size == 1 }?.first() ?: CardBrand.Unknown
```

Reply:
This causes the textfield to show an error, which we don't want